multiview/cluster/mv_k_means.py

```diff
@@ -0,0 +1,344 @@
+# Copyright 2019 NeuroData (http://neurodata.io)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Implements multi-view kmeans clustering algorithm for data with 2-views.
+
+
+import numpy as np
+from .base_cluster import BaseCluster
+from ..utils.utils import check_Xs
+from sklearn.exceptions import NotFittedError, ConvergenceWarning
+
+
+class MultiviewKMeans(BaseCluster):
+
+    '''
+    An implementation of Multi-View K-Means using the co-EM algorithm.
+    This algorithm currently handles two views of data.
+
+    Paramters
+    ---------
+    n_clusters : int (default=2)
+        The number of clusters
+
+    random_state : int (default=None)
+        Determines random number generation for initializing centroids.
+        Can seed the random number generator with an int.
+
+    patience : int, optional (default=5)
+        The number of EM iterations with no decrease in the objective
+        function after which the algorithm will terminate.
+
+    max_iter : int, optional (default=None)
+        The maximum number of EM iterations to run before
+        termination.
+
+    Attributes
+    ----------
+
+    _n_clusters : int
```

Comment:
None of your parameters should be listed as attributes here, see SKlearn API

Also unsure why these are private attributes (the leading underscore)

Reply:
Ok